bugbug/models/spambug.py

```diff
@@ -87,5 +87,14 @@ def get_labels(self):
 
         return classes, [0, 1]
 
+    def items_gen(self, classes):
+
+        for bug in bugzilla.get_bugs(include_invalid=True):
+            bug_id = bug["id"]
+            if bug_id not in classes:
+                continue
+
+            yield bug, classes[bug_id]
```

Comment:
Nit: You can rewrite this as:
`return ((bug, classes[bug["id"]]) for bug in bugzilla.get_bugs(include_invalid=True) if bug["id"] in classes)`